backends-common/elasticsearch/pom.xml

```diff
@@ -137,7 +137,7 @@
                 <dependency>
```

Comment:
JAMES-2015 make uniform version of awaitility accross the project 

Should then be your commit name